README.md

```diff
@@ -355,6 +355,16 @@ class Post
 end
 ```
 
+You can also explicitly declare which policy to use for a specific instance of a class.
+
+```ruby
+class Post
+  def policy_class
```

Comment:
Yes–that's definitely the main use case for `policy_class` @scottjacobsen, as described in the section above my addition. Defining the instance method seems more for edge cases where you want an instance to use a specific policy based on some attributes, say in a view helper `policy(post).publish?`, for example.